src/OracleService/Settings.cs

```diff
@@ -14,6 +14,16 @@ public HttpsSettings(IConfigurationSection section)
         }
     }
 
+    class NeoFSSettings
+    {
+        public string EndPoint { get; }
+
+        public NeoFSSettings(IConfigurationSection section)
+        {
+            EndPoint = section.GetValue("EndPoint", "127.0.0.1:8080");
```

Comment:
We should have a valid default value

Reply:
'127.0.0.1:8080'  is not a valid default value?

Reply:
If you will need to run your own neofs service yes.